### ISSUE TITLE
Giftsmith

### ISSUE THREAD

**Issue by shuchibooksmith** [sonnetmedia/brooklinebooksmith.com#28]
content for giftsmith will be just photos of new merchandise. we will hopefully have those photos next week along with new banner images.

**Comment by shuchibooksmith** [comment]
some inspiration for me/us for the kind of content we're looking to put up on the giftsmith page
http://www.bostongeneralstore.com/

**Comment by budparr** [comment]
Cool. Do you have that quality of photography? (hope that isn't insulting :) )

**Comment by shuchibooksmith** [comment]
funny you should ask! we're trying to figure this out right now. i think we can do some staging and take good photos. but most of the photos the giftsmith folk take is with iphones. when i read your image guidelines i couldn't quite figure out if iphone photos were...bad? i assumed yes, but now i'm not sure.

**Comment by budparr** [comment]
I'd say iPhone photos not necessarily good. They can be done well but you have to overcome a lot of hurdles. Best to use a dslr if you can swing it.

But, tada! https://www.shopify.com/blog/12206313-the-ultimate-diy-guide-to-beautiful-product-photography

if that seems useful I'll add it to the wiki

**Comment by shuchibooksmith** [comment]
yes! this is useful, thank you. can you add it to the wiki?

**Comment by shuchibooksmith** [comment]
here is the general info for the giftsmith page

The Giftsmith began in 1997 and has since grown from a small wall of greeting cards and a few quirky items to the current, beautifully designed room packed with everything from the sublime to the ridiculous. Our selection is award-winning and has helped us keep to our mission of being both an inspiring, iconic bookstore and a marvelous one-stop shopping experience. Many is the customer who remarks "I came in for a book"  or "I came in for a gift" and then happily leaves with several of both.
